File: sdkwork-backend-sdk-csharp/Models/VerificationVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class VerificationVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public bool? Success { get; set; }
        public string? Message { get; set; }
        public string? Token { get; set; }
    }
}

File: sdkwork-backend-sdk-typescript/src/types/verification-vo.ts
/** 验证响应视图对象 */
export interface VerificationVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 是否验证成功 */
  success?: boolean;
  /** 消息 */
  message?: string;
  /** 验证令牌 */
  token?: string;
}
